influxdb3_write/src/cache.rs

```diff
@@ -84,6 +86,7 @@ impl ParquetCache {
                         files.insert(
                             path.clone(),
                             ParquetFile {
+                                id: ParquetFileId::new(),
```

Comment:
There is something odd about having the cache increment the atomic counter, but I don't think it will cause any problems.